engine/menus/pokedex.asm

```diff
@@ -454,7 +454,7 @@ ShowPokedexDataInternal:
 	hlcoord 9, 2
 	call PlaceString
 
-	ld hl, PokedexEntryPointers
+	ld hl, PokedexEntryPointers ; this block sets de to PokedexEntryPointers[(wd11e-1)*2]
```

Comment:
Comments describing whole blocks/"paragraphs" of code go above the block, not next to a single line. Also, this isn't really informative until/unless `wd11e` is identified.